plugins/modules/gcp_compute_subnetwork.py

```diff
@@ -499,8 +500,33 @@ def is_different(module, response):
         if k in response:
             request_vals[k] = v
 
-    return GcpRequest(request_vals) != GcpRequest(response_vals)
-
+    if request_vals['ipCidrRange']:
+      try:
+          result_superset = cidr_superset(request_vals['ipCidrRange'], response_vals['ipCidrRange'])
```

Comment:
nit: i'd suggest something like `result_is_superset` to indicate that the value is a boolean (It's hard to infer the type of result_superset).

Reply:
Thanks @toumorokoshi for the suggestion, I changed it to `result_is_superset`.